app/services/users.js

```diff
@@ -0,0 +1,3 @@
+const userModel = require('../models').user;
```

Comment:
In general, the name for models is the same name. In this case `User`

Reply:
👍